doc/specs/#597 - Tab Sizing.md

```diff
@@ -0,0 +1,95 @@
+---
+author: Kayla Cinnamon @cinnamon-msft
+created on: 2020-01-03
+last updated: 2020-01-03
+issue id: 597
+---
+
+# Tab Sizing
+
+## Abstract
+
+This spec outlines the tab sizing feature. This is an application-level feature that is not profile-specific (at least for now).
+
+Global properties that encompass tab sizing:
+
+* `tabWidthMode` (accepts pre-defined values for tab sizing behavior)
+* `tabWidthMin` (can never be smaller than the icon width)
+* `tabWidthMax` (can never be wider than the tab bar)
+
+Acceptable values for `tabWidthMode`:
+
+* [default] `equal` (all tabs are sized the same, regardless of tab title length)
+* `titleLength` (width of tab contains entire tab title)
+
+## Inspiration
+
+Other browsers and terminals have varying tab width behavior, so we should give people options.
+
+## Solution Design
+
+`tabWidthMode` will be a global setting that will accept the following strings:
+
+* `equal`
+    * All tabs are equal in width
+    * If the tab bar has filled, tabs will shrink as additional tabs are added
+    * Utilizes the `equal` setting from WinUI's TabView
+
+* `titleLength`
+    * Tab width varies depending on title length
+    * Width of tab will fit the whole tab title
+    * Utilizes the `sizeToContent` setting from WinUI's TabView
+
+In addition to `tabWidthMode`, the following global properties will also be available:
+
+* `tabWidthMin`
+    * Accepts an integer
+    * Value correlates to the minimum amount of pixels the tab width can be
+    * If value is less than the width of the icon, the minimum width will be the width of the icon
+    * If value is greater than the width of the tab bar, the maximum width will be the width of the tab bar

```

Comment:
Isn't it somewhat weird to say the minimum width of your tab will consume the ENTIRE tab bar? Does that mean with 2 tabs you instantly get the arrows because each tab is an entire width?